app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @new_comment = Comment.new
+  end
+
+  def create
+    # @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @new_comment = @post.comments.new(comment_params)
+    @new_comment.author_id = @post.id
```

Comment:
- [ ] Let's create comment on behalf of the current user. you can use ` @comment = @current_user.comments.new(params)`

Reply:
This issue is well resolved! ✔️